report_thesis/src/sections/problem_definition.tex

```diff
@@ -1,23 +1,60 @@
 \section{Problem Definition}\label{sec:problem_definition}
+The primary objective of this research is to enhance computational methods for the accurate and robust quantification of chemical compositions using \gls{libs} data.
+This enhancement seeks to improve the adaptability and precision of \gls{libs} across various environmental contexts, including terrestrial and planetary geological samples.
 
-\subsection{Motivating Example: NASA's Mars Missions}
-The NASA Viking missions in the 1970s were the first to successfully land on Mars, aiming to determine if life existed on the planet. 
-While these missions advanced our knowledge of the Martian environment, the search for evidence of life remained inconclusive~\cite{marsnasagov_vikings}.
+Quantifying chemical compositions from \gls{libs} spectral data poses significant challenges due to the high dimensionality of the data and the complex physical and chemical interactions involved. The interpretation of \gls{libs} data is particularly complicated by factors such as multicollinearity among spectral features and matrix effects, which can obscure the relationships between the observed spectra and the actual chemical compositions.
+
+The process of elemental quantification using \gls{libs} data is constrained by several factors:
+\begin{itemize}
+    \item \textbf{Data Dimensionality and Collinearity:} High dimensionality of spectral data coupled with multicollinearity, where multiple spectral features may exhibit strong correlations, complicates the modeling and analysis\cite{andersonImprovedAccuracyQuantitative2017}.
+    \item \textbf{Matrix Effects:} Used as a catch-all term that encompasses any effect that can cause the intensity of emission lines from an element to vary independent of that element's concentration. Different background materials can alter the emission intensities and pose significant challenges in accurately quantifying elemental concentrations. The spectra are complex due to the interaction of multiple physical processes including the coupling process between the laser photons and the target, self-absorption of optical emission lines within the plasma, recombination of elements into molecules, and collisional interactions in the plasma\cite{cleggRecalibrationMarsScience2017, andersonImprovedAccuracyQuantitative2017}.
```

Comment:
```
Used as a catch-all term that encompasses any effect that can cause the intensity of emission lines from an element to vary independent of that element's concentration.
```
This sentence feels very familiar. Are we 100% sure this is not a direct copy paste error?

Reply:
Yes, it's paraphrased from `andersonImprovedAccuracyQuantitative2017`:
![image](https://github.com/chhoumann/thesis-chemcam/assets/29108628/7e24a1c6-d94d-41ac-acfc-d28b4c9a8e76)

I cited them, but do you think it's too close?

Reply:
Yes I think we need to rephrase ours. 
```suggestion
    \item \textbf{Matrix Effects:} This term refers to any effect that can cause variations in the intensity of emission lines of an element independent of the concentration of that element. Different background materials can alter the emission intensities and pose significant challenges in accurately quantifying elemental concentrations. The spectra are complex due to the interaction of multiple physical processes including the coupling process between the laser photons and the target, self-absorption of optical emission lines within the plasma, recombination of elements into molecules, and collisional interactions in the plasma\cite{cleggRecalibrationMarsScience2017, andersonImprovedAccuracyQuantitative2017}.
```

This is one possible rephrasing

Reply:
Sure!